base/ltfssaxes.dtx

```diff
@@ -192,503 +248,3158 @@
+\DeclareFontSeriesChangeRule {bex}{el}  {elex} {}
+\DeclareFontSeriesChangeRule {bex}{l}   {lex}  {}
+\DeclareFontSeriesChangeRule {bex}{sl}  {slex} {}
+\DeclareFontSeriesChangeRule {bex}{sb}  {sbex} {bex}
+\DeclareFontSeriesChangeRule {bex}{b}   {bex}  {}
+\DeclareFontSeriesChangeRule {bex}{eb}  {ebex} {bex}
+\DeclareFontSeriesChangeRule {bex}{ub}  {ubex} {bex}
+\DeclareFontSeriesChangeRule {bex}{uc}  {buc}  {}
+\DeclareFontSeriesChangeRule {bex}{ec}  {bec}  {}
+\DeclareFontSeriesChangeRule {bex}{c}   {bc}   {}
+\DeclareFontSeriesChangeRule {bex}{sc}  {bsc}  {}
+\DeclareFontSeriesChangeRule {bex}{sx}  {bsx}  {}
+\DeclareFontSeriesChangeRule {bex}{x}   {bx}   {}
+\DeclareFontSeriesChangeRule {bex}{ex}  {bex}  {}
+\DeclareFontSeriesChangeRule {bex}{ux}  {bux}  {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {bux}{ul}  {ulux} {}
+\DeclareFontSeriesChangeRule {bux}{el}  {elux} {}
+\DeclareFontSeriesChangeRule {bux}{l}   {lux}  {}
+\DeclareFontSeriesChangeRule {bux}{sl}  {slux} {}
+\DeclareFontSeriesChangeRule {bux}{sb}  {sbux} {bux}
+\DeclareFontSeriesChangeRule {bux}{b}   {bux}  {}
+\DeclareFontSeriesChangeRule {bux}{eb}  {ebux} {bux}
+\DeclareFontSeriesChangeRule {bux}{ub}  {ubux} {bux}
+\DeclareFontSeriesChangeRule {bux}{uc}  {buc}  {}
+\DeclareFontSeriesChangeRule {bux}{ec}  {bec}  {}
+\DeclareFontSeriesChangeRule {bux}{c}   {bc}   {}
+\DeclareFontSeriesChangeRule {bux}{sc}  {bsc}  {}
+\DeclareFontSeriesChangeRule {bux}{sx}  {bsx}  {}
+\DeclareFontSeriesChangeRule {bux}{x}   {bx}   {}
+\DeclareFontSeriesChangeRule {bux}{ex}  {bex}  {}
+\DeclareFontSeriesChangeRule {bux}{ux}  {bux}  {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ebuc}{ul} {uluc} {}
+\DeclareFontSeriesChangeRule {ebuc}{el} {eluc} {}
+\DeclareFontSeriesChangeRule {ebuc}{l}  {luc}  {}
+\DeclareFontSeriesChangeRule {ebuc}{sl} {sluc} {}
+\DeclareFontSeriesChangeRule {ebuc}{sb} {sbuc} {buc}
+\DeclareFontSeriesChangeRule {ebuc}{b}  {buc}  {}
+\DeclareFontSeriesChangeRule {ebuc}{eb} {ebuc} {buc}
+\DeclareFontSeriesChangeRule {ebuc}{ub} {ubuc} {buc}
+\DeclareFontSeriesChangeRule {ebuc}{uc} {ebuc} {}
+\DeclareFontSeriesChangeRule {ebuc}{ec} {ebec} {}
+\DeclareFontSeriesChangeRule {ebuc}{c}  {ebc}  {}
+\DeclareFontSeriesChangeRule {ebuc}{sc} {ebsc} {}
+\DeclareFontSeriesChangeRule {ebuc}{sx} {ebsx} {}
+\DeclareFontSeriesChangeRule {ebuc}{x}  {ebx}  {}
+\DeclareFontSeriesChangeRule {ebuc}{ex} {ebex} {}
+\DeclareFontSeriesChangeRule {ebuc}{ux} {ebux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ebec}{ul} {ulec} {}
+\DeclareFontSeriesChangeRule {ebec}{el} {elec} {}
+\DeclareFontSeriesChangeRule {ebec}{l}  {lec}  {}
+\DeclareFontSeriesChangeRule {ebec}{sl} {slec} {}
+\DeclareFontSeriesChangeRule {ebec}{sb} {sbec} {bec}
+\DeclareFontSeriesChangeRule {ebec}{b}  {bec}  {}
+\DeclareFontSeriesChangeRule {ebec}{eb} {ebec} {bec}
+\DeclareFontSeriesChangeRule {ebec}{ub} {ubec} {bec}
```

Comment:
```suggestion
\DeclareFontSeriesChangeRule {ebec}{ub} {ubec} {ebec}
```

I think it is better to use `eb` instead of `b` in the fourth argument: `eb` is a better approximation for `ub` and since the fourth argument is then identical to the first, we can assume that this font face actually exists.